packages/remix-dev/config/defaults/cloudflare/entry.server.react-stream.tsx

```diff
@@ -14,7 +14,6 @@ export default async function handleRequest(
     {
       signal: request.signal,
       onError(error: unknown) {
-        console.error(error);
```

Comment:
Now that we're doing all logging from the server I removed the logs from here.  I think this is where we got document request logging from previously and explains why it was sanitized.